server/channels/app/config_test.go

```diff
@@ -95,37 +96,42 @@ func TestEnsureInstallationDate(t *testing.T) {
 	for _, tc := range tt {
 		t.Run(tc.Name, func(t *testing.T) {
 			sqlStore := th.GetSqlStore()
-			sqlStore.GetMasterX().Exec("DELETE FROM Users")
+			_, err := sqlStore.GetMasterX().Exec("DELETE FROM Users")
+			require.NoError(t, err)
 
 			for _, createAt := range tc.UsersCreationDates {
 				user := th.CreateUser()
-				user.CreateAt = createAt
-				sqlStore.GetMasterX().Exec("UPDATE Users SET CreateAt = ? WHERE Id = ?", createAt, user.Id)
+				_, err := sqlStore.GetMasterX().Exec("UPDATE Users SET CreateAt = ? WHERE Id = ?", createAt, user.Id)
```

Comment:
Using a new variable causes linter errors as it shadows the existing variable in L99. You can just reuse it
```suggestion
				_, err = sqlStore.GetMasterX().Exec("UPDATE Users SET CreateAt = ? WHERE Id = ?", createAt, user.Id)
```

The same goes for the rest of the file.